### PR TITLE
fix: restore number camel-casing behavior changed in MR 161

### PR DESCRIPTION
**Pull request by MichaelGregory**
[MR 161](https://github.com/skovy/typed-scss-modules/pull/161) changed the behavior of how numbers are camel-cased. This MR replaces dependency change-case introduced in MR 161 with dependency [case-anything](https://github.com/mesqueeb/case-anything). case-anything is lighter weight, and configurable; although typed-scss-modules does not currently support this, it could be updated to pass configuration through to the formatter.  This MR also updates tests to demonstrate expected behavior of casing numbers.

Using selector `.color-blue-scale-10` as an example:
Legacy camel-casing transforms to `colorBlueScale10`,
MR 161 changed transform to: `colorBlueScale_10`,
This MR changes transform to `colorBlueScale10`.

case-anything has a [comprehensive test suite](https://github.com/mesqueeb/case-anything/blob/production/test/comprehensive.test.ts) demonstrating behavior.

### CODE REVIEW

**Comment by skovy** [comment]
It looks like there is also this option with `change-case`: https://github.com/blakeembrey/change-case/tree/master/packages/camel-case#merge-numbers

@MichaelGregory thoughts on using this option instead? I'd prefer to stick with the more used package

**Comment by MichaelGregory** [comment]
Both have comparable unit tests. I prefer case-anything for the smaller packages size ([1kb minified + gzipped](https://bundlephobia.com/package/case-anything@2.1.10) vs [3.5kb minified + gzipped](https://bundlephobia.com/package/switch-case@2.1.4)). If your preference remains with change-case given the higher stars, I'm happy to close or update this MR in favor of making the change to use `camelCaseTransformMerge` instead of `camelCase` from change-case :)

**Comment by skovy** [comment]
Since this is a CLI tool, none of this code should be bundled and shipped to the browser so it shouldn't have any end user-facing impact for any app/tool that uses this so I'm not too concerned about the size difference. Since this package is already being used in this repo, and it is more popular (and therefore generally more battle-tested / more likely to receive updates / etc) I'd prefer to stick with the existing `change-case` package.

Thanks for raising this regression and proposing a fix, I appreciate it! If you want to update to use `change-case` I'll happily accept this PR 👍

File: yarn.lock [hunk 1]
```diff
@@ -7250,7 +7250,12 @@ tslib@^1.9.0:
   resolved "https://registry.yarnpkg.com/tslib/-/tslib-1.14.1.tgz#cf2d38bdc34a134bcaf1091c41f6619e2f672d00"
   integrity sha512-Xni35NKzjgMrwevysHTCArtLDpPvye8zV/0E4EyYn43P7/7qvQwPh9BGkHewbMulVntbigmcT7rdX3BNo9wRJg==
 
-tslib@^2.0.3, tslib@^2.1.0, tslib@^2.2.0:
```
Comment (skovy) [inline yarn.lock:7253]: Since there was no change in `package.json` would you mind reseting these changes so there is no diff?

**Review by skovy — APPROVED** [review]
Looks great, nice work! One quick comment then I'll merge. Thanks again for raising the issue and proposing a fix 🙌

**Comment by skovy** [comment]
@all-contributors add @MichaelGregory for bug, code, test

**Pull request merged**